### ISSUE TITLE
How to integrate previous ip Duke main method with JavaFx main method?

### ISSUE THREAD

**Issue by gweeyc** [nus-tic4002-AY2122s2/forum#7]
```Java
public static void main(String[] args) {
        File f = new File(DATA_PATH);
        File fBackup = new File(BACKUP_PATH);

        if (!f.exists() && !fBackup.exists()) {
            f.getParentFile().mkdirs();
            fBackup.getParentFile().mkdirs();
            try {
                f.createNewFile();
                fBackup.createNewFile();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
        new Duke(DATA_PATH).run();

        Application.launch(args);
    }
```

Query: For the new Duke extends Application java file, I encounter the main method integration issue with the earlier ip Duke version. Obviously, there is an error at the last two lines as Duke should not run before JavaFx Application?

How then to integrate the two versions into one at main method so the program runs logically with JavaFx GUI shown and ready for user input for example?

PS: - I'm using the module-info.java configuration and including the main method in the same Duke.java file.

**Comment by damithc** [comment]
@gweeyc You can refer [this thread](https://github.com/nus-cs2103-AY2122S2/forum/issues/81) (from a different module) to locate some students who have created a similar duke GUI and see how they have done it.

**Comment by gweeyc** [comment]
HI Prof Damith, Prof Boyd,
I checked with most cases from the thread you provided, most do not have a main method in Duke.java. So I tried to take out the main method for my Duke.java, and plan to work with the Duke object created in the Main.java file as shown below:
![image](https://user-images.githubusercontent.com/43993134/152653212-092f389a-88b8-4ca6-9fa1-6e9d7983b2e6.png)


but Intellij gave this error message when I run the project? 
![image](https://user-images.githubusercontent.com/43993134/152653138-d7b2eb16-2271-4b32-8b7c-97916f0fb247.png)

Why does it call main() when I run the Launcher which points to Main.class?
![image](https://user-images.githubusercontent.com/43993134/152654063-c7446120-04e1-47e1-af3c-9a52c75f2faf.png)

_**NB: - Solved ! I found the answer, my green arrow run icon points to Duke.main() method. I just ensure I click on the green run arrow pointing to Launcher.java and I see the JavaFX GUI fine! I leave this comment here so it can remind others when they see "similar error msg" !**_

Thank you!
